PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```

Comment:
I don't have an example but maybe something like this

```suggestion
- [ ] This change requires a documentation update

## Testing steps
Please describe how the change was tested locally. If, for some reason, the testing was not done or not done fully, please describe what are the testing steps.
```

Reply:
ok good!